source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,541 @@
+All configured deployments must use the same IDP.
+
+Overview
+--------
+
+.. _minio-site-replication-what-replicates:
+
+What Replicates Across All Sites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/common-replication.rst
+   :start-after: start-mc-admin-replicate-what-replicates
+   :end-before: end-mc-admin-replicate-what-replicates
+
+What Does Not Replicate Across Sites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/common-replication.rst
+   :start-after: start-mc-admin-replicate-what-does-not-replicate
+   :end-before: end-mc-admin-replicate-what-does-not-replicate
+
+
+Initial Site Replication Process
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After enabling site replication, identity and access management (IAM) settings sync in the following order:
+
+.. tab-set::
+
+   .. tab-item:: MinIO IDP
+
+      #. Policies
+      #. User accounts (for local users)
+      #. Groups
+      #. Service accounts
+         
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+   .. tab-item:: OIDC
+
+      #. Policies
+      #. Service accounts
+         
+         Only service accounts related to user accounts with valid policies for the MinIO deployment.
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+   .. tab-item:: LDAP
+
+      #. Policies
+      #. Groups
+      #. Service accounts
+
+         Only service accounts related to user accounts with valid policies for the MinIO deployment.
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+
+Site Healing
+~~~~~~~~~~~~
+
+With site replication enabled, for any item :ref:`supported in site replication <minio-site-replication-what-replicates>`, the MinIO deployment resynchronizes that item from the peer site with the most updated ("latest") version of that item.
+
+
+Prerequisites
+-------------
+
+One Site with Data at Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Only *one* site can have data at the time of setup.
+The other sites must be empty of buckets and objects.
+
+After configuring site replication, any data  on the first deployment replicates to the other sites.
+
+.. admonition:: Wait Until Replication Completes
+   :class: warning
+
+   If there is existing data on the first peer site, wait until :mc-cmd:`mc admin replicate status` shows that the replication process has completed before performing read or write operations on the other peer sites. 
```

Comment:
not necessary to wait - if other sites are missing data, proxying will take care of reading from the first site

Reply:
Removed the admonition.